src/IO/HDF5/HDF5IOHandler.cpp

```diff
@@ -117,6 +117,11 @@ HDF5IOHandlerImpl::HDF5IOHandlerImpl(
                       << shadow << std::endl;
         }
     }
+
+    m_hdf5_collective_metadata = 0;
+    auto const hdf5_collective_metadata = auxiliary::getEnvString( "OPENPMD_HDF5_COLLECTIVE_METADATA", "OFF" );
```

Comment:
Can you please add this new option in the user-facing table in `docs/source/backends/hdf5.rst`? :)

Since `ON` provides the better performance, can you try if your CI passes if you turn this to `ON` by default?
```suggestion
    auto const hdf5_collective_metadata = auxiliary::getEnvString( "OPENPMD_HDF5_COLLECTIVE_METADATA", "ON" );
```